pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -12,6 +12,7 @@
 , widevine-cdm
 , enableVulkan ? stdenv.isLinux
 , vulkan-loader
+, isQt6 ? false
```

Comment:
Could do it automatically with something like

`isQt6 = lib.versions.major qtbase.version == "6";`
zeal and memento have that for qt5

Reply:
c3458dd757c1